process/project-lifecycle.md

```diff
@@ -0,0 +1,178 @@
+# Becoming an OpenSSF Project
+
+The OpenSSF's mission is to inspire and enable the community to secure the open source software we all depend on. OpenSSF's top priority is the security of existing open source projects however, there are missing pieces in the tooling needed to improve open source project security. These pieces are the projects OpenSSF aims to steward and support.
+
+## Project Life Cycle
+
+New projects to the OpenSSF, and progression through the project lifecycle, are approved by the Technical Advisory Council (TAC). A project's oversight is provided by either the Technical Advisory Council (TAC) or a specific Working Group (WG). When a Project reports into a specific WG, that WG can support the Project's progression and provide recommendations to the TAC.
```

Comment:
I think this text should also say:

"While projects are approved by the TAC, typically projects are first proposed to a WG for discussion and approval; the WG then raises the proposal to the TAC who determines if it's approved and the WG it will be a member of."

Reply:
+1 to David's statement